src/commonmark/en/content/developer/web-api.md

```diff
@@ -10475,13 +10475,13 @@ The analytics resource lets you specify a range of query parameters:
 <tr class="even">
 <td>outputIdScheme</td>
 <td>No</td>
-<td>Identifier scheme to use for metadata items the query response, can be identifier, code or attributes.</td>
+<td>Identifier scheme to use for metadata items the query response, can be an identifier, code or attributes.</td>
 <td>UID | CODE |NAME| ATTRIBUTE:&lt;ID&gt;</td>
 </tr>
 <tr class="odd">
 <td>inputIdScheme</td>
 <td>No</td>
-<td>Identifier scheme to use for metadata items in the query request, can be identifier, code or attributes.</td>
+<td>Identifier scheme to use for metadata items in the query request, can be an identifier, code or attributes.</td>
```

Comment:
To me this reads slightly wrongly with the additional "an" - because it cannot be _any_ identifier, it must be literally "UID".

Reply:
Yes, thanks, and I actually corrected as I read, sometimes things my slips without noticing them, thanks for the correction.